docker/devenv/Dockerfile.ubuntu.2104

```diff
@@ -0,0 +1,43 @@
+FROM ubuntu:21.04
+
+RUN apt-get update && DEBIAN_FRONTEND="noninteractive" apt-get install -y sudo \
+    && sudo apt-get update \
+    && sudo DEBIAN_FRONTEND="noninteractive" apt-get install -y \
+    apt-utils \
+    build-essential \
+    cmake \
+    flex \
+    gcc-11 \
+    g++-11 \
+    git \
+    libeigen3-dev \
+    libepoxy-dev \
+    libfontconfig1-dev \
+    libfreetype6-dev \
+    libharfbuzz-dev \
+    libogg-dev \
+    libopus-dev \
+    libopusfile-dev \
+    libpng-dev \
+    libsdl2-dev \
+    libsdl2-image-dev \
+    make \
+    ninja-build \
+    python3 \
+    python3-pip \
+    qml-module-qtquick-controls \
+    qtdeclarative5-dev \
+    && sudo apt-get clean \
+    && truncate -s 0 ~/.bash_history
+
+RUN python3 --version; \
+    python3 -m pip install --upgrade pip; \
+    python3 -m pip install \ 
+    Cython \
+    numpy \
+    jinja2 \
+    lz4 \
+    pillow \ 
+    pygments \
+    toml; \
```

Comment:
many if not all of them are regular packages in the default repos

Reply:
Hmmm what do you mean? We don't need to install them?